components/grid/selection/overview.md

```diff
@@ -10,196 +10,72 @@ position: 0
 
-## Notes
+You can get and set the selected rows or cells:
+* To access the selected rows add the Grid `SelectedItems` parameter.
+* To access the selected cells add the Grid `SelectedCells` parameter.
 
-### Editing Modes
+Both parameters are collections. You can use the parameters to manipulate the selected rows or cells. You can also use the parameters to pre-select a row or cell for your users.
 
-#### InCell Edit Mode
+See [Selected Rows]({%slug components/grid/selection/rows%}#selected-rows) and [Selected Cells]({%slug components/grid/selection/cells%}#selected-cells) for more details.
 
-In the [Incell EditMode]({%slug components/grid/editing/incell%}) selection can be applied only via a [checkbox column]({%slug components/grid/columns/checkbox%}) (`<GridCheckboxColumn />`). This is required due to the overlapping action that triggers selection and InCell editing (clicking in the row) - if row click selection was enabled with InCell editing, each attempt to select a row would put a cell in edit mode; and each attempt to edit a cell would select a new row. Such user experience is confusing, and so selection will only work through the row selection checkbox.
+## Selection Events
 
-To see how to select the row that is being edited in InCell edit mode without using a `<GridCheckboxColumn />` check out the [Row Selection in Edit with InCell EditMode]({%slug grid-kb-row-select-incell-edit%}) Knowledge Base article.
+You can respond to the user action of selecting a new item through the Grid events:
+* Use the [`SelectedItemsChanged` event]({%slug components/grid/selection/rows%}#selecteditemschanged) to respond to row selection.
+* Use the [`SelectedCellsChanged` event]({%slug components/grid/selection/cells%}#selectedcellschanged) to respond to cell selection.
 
-#### Inline and Popup Edit Modes
+> Both binding to the property and using its event cannot be used at the same time as Blazor allows only one. To use the `SelectedItemsChanged` event, utilize one-way binding for the `SelectedItems` property. Otherwise, use two-way binding for the `SelectedItems` property without the `SelectedItemsChanged` event. The same applies to the `SelectedCells` and the `SelectedCellsChanged`.
 
-In [Inline EditMode]({%slug components/grid/editing/inline%}) and [Popup EditMode]({%slug components/grid/editing/popup%}) selection can be done by clicking on the desired row or by using a `<GridCheckboxColumn />`.
+## Selection and Other Grid Features
 
-### Selection in Template
+The selection feature behavior may vary when the Grid configuration combines row or cell selection and:
+* The [Grid editing feature]({%slug components/grid/editing/overview%}).
+* The [Grid virtualization feature]({%slug components/grid/virtual-scrolling%}).
+* The [Grid paging feature]({%slug components/grid/features/paging%}).
+* The [Grid templates feature]({%slug components/grid/features/templates%}).
+* The [Grid drag and drop feature]({%slug grid-drag-drop-overview%}).
```

Comment:
These links imply that the linked pages contain information about integration with the selection feature, which they don't. You can remove the whole list and just leave the two links at the end.